bin/load-spark-env.sh

```diff
@@ -20,7 +20,29 @@
 # This script loads spark-env.sh if it exists, and ensures it is only loaded once.
 # spark-env.sh is loaded from SPARK_CONF_DIR if set, or within the current directory's
 # conf/ subdirectory.
-FWDIR="$(cd "`dirname "$0"`"/..; pwd)"
+realpath () {
```

Comment:
Nevermind, I just saw the comment about Macs not supporting that. There's no alternative on Macs? (If no, that sucks.)